src/utils/url.ts

```diff
@@ -0,0 +1,11 @@
+export function parseSearch(search: string) {
+  const params: any = {};
+  const keyValues = search.substring(1, search.length).split("&");
+
+  for (const kv of keyValues) {
+    const parts = kv.split("=");
+    params[parts[0]] = parts[1];
+  }
+
+  return params;
+}
```

Comment:
I feel like we should just use a library for this personally https://www.npmjs.com/package/qs

Reply:
We actually can just use the native `URL` web api, this was existing code I had moved

Reply:
gotcha, we don't have to swap it now just a  thought.

Reply:
@MrArnoldPalmer I'm happy to include these changes in the next PR when my prerequisite changes get merged 🙂